src/core/utils.ts

```diff
@@ -72,7 +72,7 @@ export type QueryStatusFilter = 'all' | 'active' | 'inactive' | 'none'
 
 // UTILS
 
-export const isServer = typeof window === 'undefined'
+export const isServer = typeof window === 'undefined' || 'Deno' in window
```

Comment:
Is this the best / only way to find out if you’re running in deno? What if a `'Deno'` object exists for some other reason on the window?

also, according to this stackoverflow, it seems to also be there for code run on the client if it was compiled by deno: 

https://stackoverflow.com/questions/65095009/how-to-detect-in-typescript-code-if-its-run-in-deno-or-in-browser-compiled-by-d

Reply:
What about `!!(window.Deno && Deno.version && Deno.version.deno)`?

Reply:
yeah I guess that would be safe enough